backend/src/database/attributes/member/enums.ts

```diff
@@ -0,0 +1,26 @@
+export const MemberAttributes = {
```

Comment:
Why is this file named `enums.ts` if it doesn't contain an enum?
Perhaps do something like it's done in `platforms.ts` and `platformDisplayNames.ts` files.